blueprints/ember-fullcalendar/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('fullcalendar').then(() => {
-      return this.addBowerPackageToProject('fullcalendar-scheduler').then(() => {
-        return this.addAddonToProject('ember-cli-moment-shim', '0.6.2');
-      });
+    return this.addBowerPackageToProject('fullcalendar', '^2.7.3').then(() => {
```

Comment:
Yeah, I suppose I may actually be a bit confused on how exactly packages are added. For example, in some addons, I see the bower packages in both `bower.json` and in the blueprint and in others it's just in the blueprint. Same thing for `package.json`.

As you mentioned, best practices seem hard to come by :) That said, I'm good with this.